jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientTLSTest.java

```diff
@@ -1014,6 +1013,7 @@ public void testForcedNonDomainSNI() throws Exception
             .send();
         assertEquals(HttpStatus.OK_200, response2.getStatus());
 
+        /* TODO Fix. See #6624
```

Comment:
what about marking the test method with `@DisabledOnJre({JRE.JAVA_17})` 
easier to search for common annotation `@Disabled*`

Reply:
Because the test is parameterized and tests other stuff.  So the test is not disabled, just the IPv6 part of it is.

Reply:
The problem is its hard to keep track of TODOs because we have so many of them in the code. So it might just get lost and never be fixed, maybe if you at least commented in the issue as a reminder so it is not forgotten.

Reply:
Will do.   This PR does not close the issue, so the issue will remain open.